entities/apps/businessApp.json

```diff
@@ -0,0 +1,48 @@
+{
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "number",
+            "description": "A unique identifier for the record"
+        },
+        "created_at": {
+            "type": "string",
+            "description": "The creation date and time of the entity",
+            "format": "date-time"
+        },
+        "updated_at": {
+            "type": "string",
+            "description": "The last updated date and time of the entity",
+            "format": "date-time"
+        },
+        "business_uid":{
+            "type":"string",
+            "description":"The unique identifier of the business"
+        },
+        "app_id":{
+            "type":"number",
+            "description":"The unique identifier of the app."
+        },
+        "status":{
+            "type":"string",
+            "description":"Indicates whether or not the app is installed for the business.",
+            "enum":[
```

Comment:
If these are the only status available we don't need it. install will be the create and uninstall will be the delete

Reply:
We should not delete the record. Just like with subscriptions, we need and should to keep history.